x-pack/plugins/lens/public/types.ts

```diff
@@ -180,6 +180,7 @@ export interface Datasource<T = unknown, P = unknown> {
   getDatasourceSuggestionsFromCurrentState: (state: T) => Array<DatasourceSuggestion<T>>;
 
   getPublicAPI: (props: PublicAPIProps<T>) => DatasourcePublicAPI;
+  uniqueLabels: (state: T) => Record<string, string>;
```

Comment:
We needs this for a11y of the drag/drop operation, right? Could you add a doc comment here explaining what it is and what it should return? We are missing these for all of datasource, but let's start improving now 🙂